### PR TITLE
added translation varibales to files

### PR DESCRIPTION
**Pull request by legsforwalking**
- added translation varibales to backend files
- added translations to messages.*yaml
- more commits will follow

### CODE REVIEW

File: src/Mapbender/CoreBundle/Element/Type/CoordinatesDisplayAdminType.php [hunk 1]
```diff
@@ -15,6 +15,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('numDigits', 'Symfony\Component\Form\Extension\Core\Type\IntegerType', array(
                 'required' => true,
+                'label' => 'mb.core.coordinatesdesplay.admin.numdigits',
             ))
             ->add('label', 'Symfony\Component\Form\Extension\Core\Type\CheckboxType', array(
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Element/Type/CoordinatesDisplayAdminType.php:20]: Why label is not translated

File: src/Mapbender/CoreBundle/Element/Type/FeatureInfoAdminType.php [hunk 2]
```diff
@@ -14,6 +14,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('displayType', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array(
                 'required' => true,
+                'label' => 'mb.core.featureinfo.admin.displaytype',
                 'choices' => array(
                     'Tabs' => 'tabs',
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Element/Type/FeatureInfoAdminType.php:19]: Translationf for Tabs and Accordion is missing

File: src/Mapbender/CoreBundle/Element/Type/SrsSelectorAdminType.php [hunk 3]
```diff
@@ -14,6 +14,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('tooltip', 'Symfony\Component\Form\Extension\Core\Type\TextType', array(
                 'required' => false,
+                'label' => 'mb.core.srsselector.admin.srsselector',
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Element/Type/SrsSelectorAdminType.php:17]: must be 'label' => 'mb.core.srsselector.admin.tooltip',

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 4]
```diff
@@ -66,6 +71,8 @@ mb:
         help: Hilfe
         info: Info
         about: Info-Dialog
+      admin:
+        tooltip: About
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:75]: About -> Über

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 5]
```diff
@@ -213,6 +225,11 @@ mb:
         gps: GPS
         position: Position
         button: Button
+      admin:
+        average: Durchschnitt
+        follow: Folge
+        centeronfirstposition: Zentrieren auf der ersten Position
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:231]: Zentriere auf die erste Position

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 6]
```diff
@@ -213,6 +225,11 @@ mb:
...
+        zoomtoaccuracyonfirstposition: Zoom auf Genauigkeit bei der ersten Position
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:232]: Puh. Bin mir nicht sicher. aber besser:
Zoom auf Genauigkeit der ersten Position

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 7]
```diff
@@ -254,22 +271,40 @@ mb:
         control: Kontrolle
         navigation: Navigation
         panel: Panel
+      admin:
+        components: Komponenten
+        rotation: Drehungen
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:276]: Drehen

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 8]
```diff
@@ -254,22 +271,40 @@ mb:
...
+        zoommax: Zoom auf maximale Ausdehnung
+        zoominout: Rein- und rauszoomen
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:278]: Rein- und Rauszoomen

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 9]
```diff
@@ -254,22 +271,40 @@ mb:
...
       class:
         title: Aktivitätsanzeige
         description: Zeigt HTTP-Aktivität an.
       tag:
         activity: Aktivität
         indicator: Indikator
+      admin:
+        tooltip: Tooltip
+        activityclass: Aktivitätsklasse
+        ajaxactivityclass: Ajax
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:290]: Ajax-Aktivitätsklasse
Kachel-Aktivitätsklasse
-> please check @Phocacius

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 10]
```diff
@@ -329,13 +369,17 @@ mb:
...
+        tooltip: Tooltip
     srsselector:
       class:
         title: SRS Auswahl
         description: Nach der Auswahl eines räumlichen Referenzsystems (SRS) ändert sich das SRS in der Karte.
       tag:
         srs: SRS
         selector: Auswahl
+      admin:
+        srsselector: SSR-Wähler
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:382]: SRS-Auswahl -> not ssr

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 11]
```diff
@@ -391,6 +435,9 @@ mb:
           centermap: Karte zentrieren
       backend:
         addMapSrsList: Koordinatensysteme von der Karte hinzufügen
+      admin:
+        srslist: SRS Liste
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:439]: SRS-Liste

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 12]
```diff
@@ -448,6 +495,9 @@ mb:
       class:
         title: HTML
         description: HTML hinzufügen
+      admin:
+        content: Inhalf
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:499]: Inhalt

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 13]
```diff
@@ -490,6 +541,16 @@ mb:
         description: Wechselt unter Beibehaltung der aktuellen Kartenposition zu einer anderen Anwendung
       admin:
         open_in_new_tab: In neuem Tab öffnen
+    coordinatesdesplay:
+      admin:
+        numdigits: Zahlenziffern
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:546]: Anzahl der Nachkommastellen

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 14]
```diff
@@ -490,6 +541,16 @@ mb:
...
+        empty: Leer
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:547]: Leere Anzeige

File: src/Mapbender/CoreBundle/Resources/translations/messages.en.yaml [hunk 15]
```diff
@@ -252,22 +269,40 @@ mb:
...
       class:
         title: Activity indicator
         description: Shows HTTP activity
       tag:
         activity: activity
         indicator: indicator
+      admin:
+        tooltip: Tooltip
+        activityclass: Activity class
+        ajaxactivityclass: Ajax
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.en.yaml:288]: Ajax Activity Class
Tile Activity Class

File: src/Mapbender/CoreBundle/Resources/translations/messages.en.yaml [hunk 16]
```diff
@@ -488,6 +539,16 @@ mb:
         description: Switches to another application while maintaining current map position
       admin:
         open_in_new_tab: Open in new tab
+    coordinatesdesplay:
+      admin:
+        numdigits: Number Digits
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.en.yaml:544]: Number of decimal places

**Review by astroidex — CHANGES_REQUESTED** [review]

File: src/Mapbender/CoreBundle/Element/Type/CoordinatesDisplayAdminType.php (hunk 1 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Element/Type/CoordinatesDisplayAdminType.php:20]: Fixed :)

File: src/Mapbender/CoreBundle/Element/Type/FeatureInfoAdminType.php (hunk 2 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Element/Type/FeatureInfoAdminType.php:19]: Fixed :)

File: src/Mapbender/CoreBundle/Element/Type/ButtonAdminType.php
[the remark this answers is not in the document]
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Element/Type/ButtonAdminType.php:1]: fixed :)

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 13 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:546]: fixed

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 14 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:547]: fixed

File: src/Mapbender/CoreBundle/Resources/translations/messages.en.yaml (hunk 15 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.en.yaml:288]: fixed

File: src/Mapbender/CoreBundle/Resources/translations/messages.en.yaml (hunk 16 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.en.yaml:544]: fixed

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 12 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:499]: fixed

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 5 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:238]: fixed.

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 6 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:239]: fixed

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 7 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:289]: fixed

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 11 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:458]: fixed

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 8 above)
Comment (legsforwalking) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:291]: fixed

File: src/Mapbender/CoreBundle/Element/Type/MapAdminType.php [hunk 17]
```diff
@@ -52,10 +53,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ))
             ->add('tileSize', NumberType::class, array(
                 'required' => false,
-                'label' => 'Tile size',
+                'label' => 'mb.core.map.admin.tilesize',
             ))
             ->add('srs', TextType::class, array(
-                'label' => 'SRS',
+                'label' => 'mb.core.map.admin.SRS',
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Element/Type/MapAdminType.php:59]: srs in lower case

File: src/Mapbender/CoreBundle/Element/Type/OverviewAdminType.php [hunk 18]
```diff
@@ -29,6 +29,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('layerset', 'Mapbender\CoreBundle\Element\Type\LayersetAdminType', array(
                 'application' => $options['application'],
                 'required' => true,
+                'label' => 'mb.core.overview.admin.layerset',
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Element/Type/OverviewAdminType.php:32]: wrong: 'mb.core.overview.admin.layerset',
should be: 'mb.core.layerset.admin.layerset',

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 19]
```diff
@@ -144,6 +157,10 @@ mb:
         description: Element über das eine Treffpunkt generiert und verschickt werden kann.
       label:
         text: Bearbeiten Sie den Text
+      admin:
+        body: Körper
+        gps: gps
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:162]: GPS in upper case

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 20]
```diff
@@ -303,13 +353,21 @@ mb:
       srsnotfound: SRS Eigenschaften für %srslist% wurden nicht gefunden.
       admin:
         fixedZoomSteps: Feste Maßstabsstufen
+        layersets: Layersets
+        tilesize: Kachelgröße
+        SRS: SRS
+        scales: Maßstäbe (csv)
```
Comment (astroidex) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:359]: Maßstäbe (komma-separiert)

File: src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml [hunk 21]
```diff
@@ -53,10 +53,21 @@ mb:
...
+        templates: Vorlagen
+        qualitylevels: Qualitätsstufen
+      label:
+        quality: Qualität
+      class:
+        title: Titel
     printclienttemplate:
       admin:
         template: Vorlage
         label: Label
```
Comment (astroidex) [inline src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml:65]: Should be Titel or Beschriftung in the german Version

File: src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml [hunk 22]
```diff
@@ -53,10 +53,21 @@ mb:
...
     printclientquality:
       admin:
         label: Label
```
Comment (astroidex) [inline src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml:68]: Should be Titel or Beschriftung in the german Version

**Review by astroidex — CHANGES_REQUESTED** [review]

File: src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml [hunk 23]
```diff
@@ -162,3 +168,36 @@ mb:
...
+    dimensionshandler:
+      admin:
+        tooltip: Tooltip
+        dimensionsets: Dimensionssets
+    wmsloader:
+      admin:
+        defaultformat: Standardformat
+        defaultinfoformat: Standard-Infoformat
+    admin:
+      ? ''
```
Comment (Phocacius) [inline src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml:186]: This yields an error when starting the application:

`An exception has been thrown during the rendering of a template ("The file "/var/www/mapbender/application/mapbender/src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml" does not contain valid YAML: Complex mappings are not supported at line 184 (near "? ''").").`

Same in english version

File: src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml [hunk 24]
```diff
@@ -162,3 +168,36 @@ mb:
...
+          default: Default
+          hidden: Versteckt
+      onlineresource:
+        format: Format
+        href: href
+      legendurltype:
+        width: Breite
+        height: Höhe
+        onlineresource: Online-Ressource
+label' => 'mb:
```
Comment (Phocacius) [inline src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml:198]: this looks like an error, is this really the label you wanted to have?

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 25]
```diff
@@ -144,6 +157,10 @@ mb:
         description: Element über das eine Treffpunkt generiert und verschickt werden kann.
       label:
         text: Bearbeiten Sie den Text
+      admin:
+        body: Körper
```
Comment (Phocacius) [inline src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:161]: "Körper" sagt man in dem Kontext nicht auf deutsch. "Inhalt" oder einfach "Text" würde es besser treffen

File: src/Mapbender/WmsBundle/Form/Type/VendorSpecificType.php [hunk 26]
```diff
@@ -30,6 +30,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('vstype', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array(
+                'label' => 'mb.core.admin..vendorspecifictype.vstype',
```
Comment (Phocacius) [inline src/Mapbender/WmsBundle/Form/Type/VendorSpecificType.php:33]: Double dot, translation can't be shown (for all in this file)

**Review by Phocacius — CHANGES_REQUESTED** [review]
Missing translations now:

- Icons (application/mapbender/src/Mapbender/CoreBundle/Component/IconPackageFa4.php and application/mapbender/src/Mapbender/CoreBundle/Component/IconPackageMbIcons.php)
- The "title" added to all elements: application/mapbender/src/Mapbender/ManagerBundle/Component/ElementFormFactory.php#78

Otherwise, thank you for your work!

File: src/Mapbender/CoreBundle/Element/Type/BaseSourceSwitcherAdminType.php [hunk 27]
```diff
@@ -39,6 +41,7 @@ public function preSetData(FormEvent $event)
             $event->getForm()
                 ->add('instancesets', SortableCollectionType::class, array(
                     'entry_type' => 'Mapbender\CoreBundle\Element\Type\InstanceSetAdminType',
+                    'label' => 'mb.core.basesourceswitcher.admin.instances',
```
[the remark this answers is not in the document]
Comment (astroidex) [reply src/Mapbender/CoreBundle/Element/Type/BaseSourceSwitcherAdminType.php:44]: still open

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 19 above)
Comment (astroidex) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:162]: still open

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 25 above)
Comment (astroidex) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml]: Text passt doch gut

File: src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml (hunk 21 above)
Comment (astroidex) [reply src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml]: still open

File: src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml (hunk 22 above)
Comment (astroidex) [reply src/Mapbender/PrintBundle/Resources/translations/messages.de.yaml]: still open

File: src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml [hunk 28]
```diff
@@ -68,6 +68,12 @@ mb:
             layerOrder: Layer-Reihenfolge
             layerOrder.standard: Standard (umgekehrt)
             layerOrder.reverse: QGIS Style (gleiche Reihenfolge)
+            exceptionformat: Fehlerformat
+            infoformat: Format der GetFeatureInfo-Anfrage
+            format': Format der GetMap-Anfrage
+            dimensions: Abmessungen
+            vendorspecifics: Vendor specifics
```
[the remark this answers is not in the document]
Comment (astroidex) [reply src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml:75]: looks good

File: src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml [hunk 29]
```diff
@@ -162,3 +168,36 @@ mb:
...
+        tooltip: Tooltip
+        dimensionsets: Dimensionssets
+    wmsloader:
+      admin:
+        defaultformat: Standardformat
+        defaultinfoformat: Standard-Infoformat
+    admin:
+      ? ''
+      : vendorspecifictype:
+          name: Namen
```
Comment (astroidex) [inline src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml]: : vendorspecifictype: -> can not be correct : before 

Name nicht Namen, oder?

File: src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml [hunk 30]
```diff
@@ -162,3 +168,36 @@ mb:
...
+        format: Format
+        href: href
+      legendurltype:
+        width: Breite
+        height: Höhe
+        onlineresource: Online-Ressource
+label' => 'mb:
+  core:
+    admin:
+      ? ''
```
Comment (astroidex) [inline src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml]: ? ' ' error

also 
: vendorspecifictype: -> can not be correct : before

File: src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml [hunk 31]
```diff
@@ -68,6 +68,12 @@ mb:
             layerOrder: Layer ordering
             layerOrder.standard: standard
```
Comment (astroidex) [inline src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml:69]: - please fix: upper case to lower case
- . in the variable
            layerOrder.standard: standard
            layerOrder.reverse: QGIS

File: src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml [hunk 32]
```diff
@@ -68,6 +68,12 @@ mb:
...
             layerOrder.reverse: QGIS
+            exceptionformat: Exception format
+            infoformat: Format of GetFeatureInfo request
+            format': Format of the GetMap request
```
Comment (astroidex) [inline src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml:73]: format' -> error '

File: src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml [hunk 33]
```diff
@@ -68,6 +68,12 @@ mb:
...
+            dimensions: Dimensions
+            vendorspecifics: Vendor specifics
+            layers: Layers
           vendorspecific:
             label: Vendor specific
```
Comment (astroidex) [inline src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml:78]: specific or specifics what makes sense?

File: src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml [hunk 34]
```diff
@@ -160,3 +166,36 @@ mb:
...
+    dimensionshandler:
+      admin:
+        tooltip: Tooltip
+        dimensionsets: Dimensionsets
+    wmsloader:
+      admin:
+        defaultformat: Default format
+        defaultinfoformat: Default info format
+    admin:
+      ? ''
```
Comment (astroidex) [inline src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml:184]: ? ' ' error

also
: vendorspecifictype: -> can not be correct : before

File: src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml [hunk 35]
```diff
@@ -160,3 +166,36 @@ mb:
...
+          default: Default
+          hidden: Hidden
+      onlineresource:
+        format: Format
+        href: href
+      legendurltype:
+        width: Width
+        height: Height
+        onlineresource: Online resource
+label' => 'mb:
```
Comment (astroidex) [inline src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml:205]: wrong syntax

label' => 'mb:
  core:
    admin:
      ? ''
      : vendorspecifictype:

**Review by astroidex — CHANGES_REQUESTED** [review]

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml (hunk 25 above)
Comment (Phocacius) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml]: nein. Hier geht es um den HTML-Inhalt, der beim Senden der Nachricht eines POI gesendet wird, das heißt im Deutschen nicht Körper.

File: src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml (hunk 35 above)
Comment (Phocacius) [reply src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml:205]: that is outdated already

File: src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml [hunk 36]
```diff
@@ -490,6 +567,26 @@ mb:
...
+        title: Titel
+        group: Gruppe
+        instances: Instanzen
+    basebutton:
+      admin:
+        tooltip: Tooltip
+        icon: Icon
   template:
     toolbar_menu_tooltip: Menü öffnen
     region:
```
[the remark this answers is not in the document]
Comment (Phocacius) [reply src/Mapbender/CoreBundle/Resources/translations/messages.de.yaml:599]: No, this opens a group (with toolbar, footer, sidepant and content as children, and they are translated)

**Pull request merged**